README.md

```diff
@@ -61,53 +62,107 @@ Versioning is not supported at this time.
-      }
+    // Defines how the S3 API server connects to an iRODS server.
+    "irods_client": {
+        "host": "<string>",
+        "port": 1247,
+
+        "zone": "<string>",
+
+        "rodsadmin": {
+            "username": "<string>",
+            "password": "<string>"
+        }
     },
-    "static_authentication_resolver": {
-      "name": "static_authentication_resolver",
-      "users": {
-        "<username>": {// You log into the s3 with *this* username
-          "username": "rods",// This is the irods username
-          "secret_key": "<secret key>" // This is the secret key on the s3 side
+
+    // Defines the set of plugins to load.
+    "plugins": {
+        //
+        // Each key corresponds to a plugin's .so file name, minus the "lib" prefix.
+        //
+
+        "static_bucket_resolver": {
+            // The internal name assigned to the plugin.
+            "name": "static_bucket_resolver",
+
+            // Defines the mapping between bucket names and iRODS collections.
+            "mappings": {
+                "<bucket_name>": "/path/to/collection"
+            }
+        },
+
+        "static_authentication_resolver": {
+            // The internal name assigned to the plugin.
+            "name": "static_authentication_resolver",
+
+            // Defines information for resolving an S3 username to an iRODS username.
+            "users": {
+                // Maps <s3_username> to a specific iRODS user.
+                // Each iRODS user that intends to access the S3 API must have at least one entry.
+                "<s3_username>": {
+                    // The iRODS username to resolve to.
+                    "username": "<string>",
+
+                    // The secret key used to authenticate with the S3 API for this user.
+                    "secret_key": "<string>"
+                }
+            }
         }
-      }
-    }
-  },
-  // This may be relevant to your performance if you have many rules in your iRODS installation
-  "resource": "The_Fastest_iRODS_Resource",
-  // This is automatically set to 3 times the number of processors if this value is not specified.
-  "threads": 10
+    },
+
+    // This may be relevant to your performance if you have many rules in your iRODS installation
+    "resource": "the_fastest_irods_resource",
+
+    // The number of threads dedicated to servicing client requests.
+    // This is automatically set to 3 times the number of processors if this value is not specified.
+    "threads": 10
 }
 ```
 
-When an S3 request passes through to iRODS, it becomes a connection to the
-irods server which is pretending to be the user that it is mapped to.
-This means that you are limited to what you can do there, as an irods user,
-rather than whatever expectation might exist with a proper S3 environment.
+## Running
+
+Docker is required for running the S3 API server. To do so, run the following:
```

Comment:
I don't think this is true... but this is how we're doing it until we get it fully packaged?

Will discuss.

Reply:
Docker is a temporary requirement because we don't ship Boost 1.81.

Reply:
ah, okay.

so we'll tweak this language later once we can have them just 'install' and run the binary alongside a config file.

Reply:
Right.